server.js

```diff
@@ -1,49 +1,41 @@
-let categories = require('./my-app-react/src/data/categories.json')
-let products = require('./my-app-react/src/data/products.json')
+let dataService = require('./dataservice.js');
 
-const express = require('express')
-const app = express()
+const express = require('express');
+const app = express();
 
 app.get('/info', (req, res) => {
-    info = [
-        {"serverName": "localhost5000"},
+    let info = [
+        {"serverName": "localhost5001"},
         {"sampleName": "chinemerem-react"},
```

Comment:
Did you address my last comment?
```suggestion
        {"serverName": "chinemerem-react"},
```
Remove `sampleName`.